generateCarousel.js

```diff
@@ -0,0 +1,57 @@
+import { fetchData } from './getData.js'
+import { carouselNavButtonsEventListeners } from './script.js'
+
+function generateCarouselPhotos(item, index) {
+	const prevButton = document.getElementById('left-btn')
```

Comment:
It would probably be better to have an element (container) with some sort of id to with all of the carousel items would be appended.